test/e2e/cloud/cluster_upgrade.go

```diff
@@ -138,7 +138,7 @@ var _ = SIGDescribe("Upgrade [Feature:Upgrade]", func() {
 			framework.ExpectNoError(err)
 
 			testSuite := &junit.TestSuite{Name: "Cluster upgrade"}
-			clusterUpgradeTest := &junit.TestCase{Name: "[sig-cluster-lifecycle] cluster-upgrade", Classname: "upgrade_tests"}
+			clusterUpgradeTest := &junit.TestCase{Name: "[sig-cloud-provider-gcp] cluster-upgrade", Classname: "upgrade_tests"}
```

Comment:
Upgrade tests seem like cluster lifecycle to me, are we moving because the test is GCP specific?

Reply:
I believe something that we should mention here is that a lot of these tests rely on the cluster directory stuff. We have been moving a lot of the unmaintained things that rely on cluster/ to sig-gcp (dormant group) because some of the prominent parts of cluster/ are gcp specific

Reply:
> Upgrade tests seem like cluster lifecycle to me, are we moving because the test is GCP specific?

yes, apart from @justinsb nobody in the SIG knows how to debug these.